<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="Button">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>160</y>
      <width>75</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>启动</string>
    </property>
   </widget>
   <widget class="QLabel" name="BigCar">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>大车</string>
    </property>
   </widget>
   <widget class="QLabel" name="SmartCar">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>120</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>小车</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="BigCarCount">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Simplified Arabic</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="midLineWidth">
     <number>1</number>
    </property>
    <property name="numDigits">
     <number>6</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="SmartCarCount">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Simplified Arabic</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="midLineWidth">
     <number>1</number>
    </property>
    <property name="numDigits">
     <number>6</number>
    </property>
   </widget>
   <widget class="QLabel" name="sum">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>170</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>总计</string>
    </property>
   </widget>
   <widget class="QLabel" name="DateAndTime">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>391</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLCDNumber" name="sumCount">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>160</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Simplified Arabic</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="midLineWidth">
     <number>1</number>
    </property>
    <property name="numDigits">
     <number>6</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
